integration/utilities/kube/kube.go

```diff
@@ -26,6 +28,7 @@ func NewTest(kubeconfigPath string) (*harness.Test, error) {
 		return nil, err
 	}
 	test := h.NewTest(t)
+	test.Namespace += fmt.Sprintf("-%d", t.ParallelProcess())
```

Comment:
This will change the namespace in other tests, which should not cause any failures, but for now, we should do this only if the test is being run with `-p`.